modules/microshift-install-rhel-image-mode-cni.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-run-image-cni_{context}"]
+= Checking the {microshift-short} bootc image for the CNI
+
+Check for and workaround a kernel mismatch to enable the {microshift-short} OVN-Kubernetes (OVN-K) Container Network Interface (CNI).
+
+[IMPORTANT]
+====
+The {microshift-short} OVN-K CNI requires the `openvswitch` kernel module to be available in the bootc image. A bootc image is started as a container that uses the host kernel. The host kernel might be different from the one used for building the image, resulting in a mismatch. A kernel version mismatch with the modules present in the `/lib/modules` directory means that the `openvswitch` module cannot be loaded in the container and the container fails to run. You can work around this problem by pre-loading the `openvswitch` module.
+====
+
+.Prerequisites
+
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have `sudo` permissions.
+
+.Procedure
+
+. Check the kernel version for the `openvswitch` module by running the following command:
++
+[source,terminal]
+----
+$ find /lib/modules/$(uname -r) -name "openvswitch*"
+----
++
+.Example output
+[source,terminal]
+----
+/lib/modules/6.9.9-200.fc40.x86_64/kernel/net/openvswitch
+/lib/modules/6.9.9-200.fc40.x86_64/kernel/net/openvswitch/openvswitch.ko.xz
```

Comment:
This output is from Fedora kernel, why?

Reply:
It's from the [enhancement](https://github.com/openshift/microshift/pull/3690/files#diff-c88c33f18fdab6734ddf0f4435017f1345c4f8e06ece18b292b145fdeba9791bR148). I don't have a way to show another output. Can you provide a RHEL output?

Reply:
Ah, here's the output:
```
$ find /lib/modules/$(uname -r) -name "openvswitch*"
/lib/modules/5.14.0-427.28.1.el9_4.x86_64/kernel/net/openvswitch
/lib/modules/5.14.0-427.28.1.el9_4.x86_64/kernel/net/openvswitch/openvswitch.ko.xz
```